src/data/registration/registration_rollback.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
+
+# This script rolls back the registration status on the server
+
+ROOT=${1:-/}
+
+# print the progress in green so it is better visible
+echo -e "\e[0;32mRestoring the registration status at ${ROOT}, \
+this might take several minutes...\e[0m"
+
+# run the rollback
+chroot "$ROOT" SUSEConnect --rollback
```

Comment:
will it really work?in my testing instsys I usually have mount bind /dev. Also what happen when installation abort when /mnt is already unmounted or not yet mounted?

Reply:
Yes, it works, see the screencasts. I just have retested it with the updated packages.

When `/mnt` is not mounted then the scripts are not found and not executed (they are stored in `/mnt/var/adm/backup/system-upgrade`). If the target is not mounted we cannot do anything about it anyway.

See https://github.com/yast/yast-installation/blob/0c251710807ee463cf4df2a0b20c875675f9ded4/startup/YaST2.call#L279-L285